plugins/woocommerce-blocks/assets/js/blocks/cart-checkout-shared/block-settings/express-payment-settings.tsx

```diff
@@ -0,0 +1,190 @@
+		select( PAYMENT_STORE_KEY ).getAvailableExpressPaymentMethods();
+
+	return Object.values( availableExpressMethods ).reduce(
+		( acc, currentValue ) => {
+			return (
+				acc ||
+				currentValue?.supportsStyle.some( ( el ) =>
+					styleControl.includes( el )
+				)
+			);
+		},
+		false
+	);
+};
+
+const ExpressPaymentButtonStyleControls = ( {
+	attributes,
+	setAttributes,
+}: {
+	attributes: BlockAttributes;
+	setAttributes: ( attrs: BlockAttributes ) => void;
+} ) => {
+	const { buttonHeight, buttonBorderRadius } = attributes;
+
+	return (
+		<>
+			{ atLeastOnePaymentMethodSupportsOneOf( [ 'height' ] ) && (
+				<RadioControl
+					label={ __( 'Button height', 'woocommerce' ) }
+					selected={ buttonHeight }
+					options={ [
+						{ label: 'Small (40px)', value: '40' },
+						{ label: 'Medium (48px)', value: '48' },
+						{ label: 'Large (55px)', value: '55' },
+					] }
+					onChange={ ( newValue: string ) =>
+						setAttributes( { buttonHeight: newValue } )
+					}
+				/>
+			) }
+			{ atLeastOnePaymentMethodSupportsOneOf( [ 'borderRadius' ] ) && (
+				<div className="border-radius-control-container">
+					<HeightControl
+						label={ __( 'Button border radius', 'woocommerce' ) }
+						value={ buttonBorderRadius }
+						onChange={ ( newValue: string ) => {
+							const valueOnly = newValue.replace( 'px', '' );
+							setAttributes( {
+								buttonBorderRadius: valueOnly,
+							} );
+						} }
+					/>
+				</div>
+			) }
+		</>
+	);
+};
+
+const ExpressPaymentToggle = ( {
+	attributes,
+	setAttributes,
+}: {
+	attributes: BlockAttributes;
+	setAttributes: ( attrs: BlockAttributes ) => void;
+} ) => {
+	if ( attributes.showButtonStyles ) {
+		return (
+			<ExpressPaymentButtonStyleControls
+				attributes={ attributes }
+				setAttributes={ setAttributes }
+			/>
+		);
+	}
+	return null;
+};
+
+const ExpressPaymentMethods = () => {
+	const availableExpressMethods =
+		select( PAYMENT_STORE_KEY ).getAvailableExpressPaymentMethods();
+
+	if ( Object.entries( availableExpressMethods ).length < 1 ) {
+		return (
+			<p className="wc-block-checkout__controls-text">
+				{ __(
+					'You currently have no express payment integrations active.',
+					'woocommerce'
+				) }
+			</p>
+		);
+	}
+
+	return (
+		<>
+			<p className="wc-block-checkout__controls-text">
+				{ __(
+					'You currently have the following express payment integrations active.',
+					'woocommerce'
+				) }
+			</p>
+			{ Object.values( availableExpressMethods ).map( ( values ) => {
+				return (
+					<ExternalLinkCard
+						key={ values.name }
+						href={ `${ ADMIN_URL }admin.php?page=wc-settings&tab=checkout&section=${ encodeURIComponent(
+							values.gatewayId
+						) }` }
+						title={ values.title }
+						description={ values.description }
+					/>
+				);
+			} ) }
+		</>
+	);
+};
+
+const toggleLabel = (
+	<>
+		{ __( 'Apply uniform styles', 'woocommerce' ) }{ ' ' }
+		<span className="express-payment-styles-beta-badge">Beta</span>
```

Comment:
I don't know if this has been used anywhere else, but me and Nikki thought it works quite well as a subtle way to communicate that it's not fully integrated with all payment extensions yet.

Reply:
This is how it was used in Gutenberg for the global styles, I don't have strong feelings toward the setup, but it feels too heavy with the label, and I think it might make sense in the panel title?

![image](https://github.com/user-attachments/assets/f4a0f241-d3da-4212-ad47-0fdddb5080d9)

Reply:
This was the CSS for it:

```css
.edit-site-navigation-sidebar__beta {
	display: inline-flex;
	margin-left: $grid-unit-10;
	padding: 0 $grid-unit-10;
	height: $grid-unit-30;
	border-radius: $radius-block-ui;
	background-color: $gray-900;
	color: $white;
	align-items: center;
	font-size: $helptext-font-size;
	line-height: 1;
}
```